mm2src/coins/utxo/utxo_common_tests.rs

```diff
@@ -154,26 +155,27 @@ pub(super) async fn wait_for_tx_history_finished<Coin>(
 where
     Coin: CoinWithTxHistoryV2 + MmCoin,
 {
-    let started_at = now_ms() / 1000;
-    let wait_until = started_at + timeout_s;
-
     let req = MyTxHistoryRequestV2 {
         coin: coin.ticker().to_owned(),
         limit: u32::MAX as usize,
         paging_options: PagingOptionsEnum::PageNumber(NonZeroUsize::new(1).unwrap()),
         target,
     };
 
-    while now_ms() / 1000 < wait_until {
-        Timer::sleep(3.).await;
+    // Let the storage to be initialized for the given coin.
+    Timer::sleep(1.).await;
```

Comment:
you changed the time?? why tho?

Reply:
I've left the sleep timeout between attempts the same - 3 seconds as before:
```rust
.repeat_every(Duration::from_secs(3))
```

But as you can see (before my changes), the sleep timer was at the beginning of this function (i.e. before the first attempt). It's required to let the storage to initialize everything it needs before the first `my_tx_history_v2_impl` attempt.
`repeatable` doesn't allow to sleep before the first attempt, so I decided to put a sleep at the beginning of this function.
It's enough to sleep for 1 second to let the storage to initialize all required SQL tables.

Reply:
gotcha 👌🏼